### ISSUE TITLE
CSV importer should check for negative numbers

### ISSUE THREAD

**Issue by michieldenhaan** [quintel/etlocal#164]
Currently it is possible to add negative numbers (and maybe non-numeric values as well?) to the database using the CSV import feature. This causes datasets to break when exporting to ETSource. It would be nice if the importer throws an error when the CSV contains negative/non-numeric values (except for the `geo_id` and `name` fields which do contain non-numeric values)

**Comment by mabijkerk** [comment]
@noracato do we even still have a CSV importer feature?

**Comment by noracato** [comment]
Yes, the CSV importer is used for all the data migrations. I think it wouldn't hurt to add some extra validators to it at one point.
